doc/api/fs.md

```diff
@@ -1285,6 +1286,7 @@ changes:
   * `mode` {integer}
   * `autoClose` {boolean}
   * `start` {integer}
+* Returns: {WriteStream}
```

Comment:
Should be `{stream.Writable}`
See [type-parser.js#L81-L84](https://github.com/nodejs/node/blob/0e7b61229aa602e55c5fb034a63d7da97eecff3b/tools/doc/type-parser.js#L81-L84)